.github/workflows/static_checks_etc.yml

```diff
@@ -33,6 +33,12 @@ jobs:
         if: steps.skip-workflow.outputs.skip-workflow == 'false'
         uses: actions/checkout@v3
 
+      - name: Run FOSSA scan and upload build data
+        if: github.ref == 'refs/heads/main'
```

Comment:
This was required, at least for now, for the pull request action to pass:
  - https://github.com/fossa-contrib/fossa-action#push-only-api-token
  - https://docs.fossa.com/docs/api-reference#push-only-api-token